lychee-lib/src/extract.rs

```diff
@@ -125,6 +132,8 @@ fn elem_attr_is_link(attr_name: &str, elem_name: &str) -> bool {
     // over at: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
     matches!(
         (attr_name, elem_name),
+        // TODO: re-enable once `or-patterns` become stable in Rust 1.53
+        // ("href" | "src" | "srcset" | "cite", _) | ("data", "object") | ("onhashchange", "body")
         ("href", _)
```

Comment:
When I refactored this part, it's requested to keep the current form for readability.